### ISSUE TITLE
[Bug]: Community Tools item order

### ISSUE THREAD

**Issue by cupangafk** [neon-nyan/Collapse#189]
### Affected Version

1.70.11

### Bug Behavior

first of all
this is not really a bug, but i dont know what kind of issue i need to make

item order in the community tools (official tools to be exact) need to be consistent
![image](https://github.com/neon-nyan/Collapse/assets/66926348/0da5ee79-6cf9-4df8-a084-53093519a1c0)
![image](https://github.com/neon-nyan/Collapse/assets/66926348/078b9f44-1e80-41cd-bb02-0ed7044e15e6)
![image](https://github.com/neon-nyan/Collapse/assets/66926348/c4711be7-eeae-472f-b925-981938ca9169)

it doesnt hurt anyone but my muscle memory 

### Expected Behavior

-

### Steps to reproduce

-

**Comment by bagusnl** [comment]
Hello~

Sorry for the inconvenience about the Community Tools ordering. We will work on that for our next release.
In the meantime, you can use the attached CommunityTools.json file.

How to apply:
- Go to your Collapse installation folder, then go to app-x-xx-xx (pick the latest one), then Assets > Preset.
- Open CommunityTools.json then replace everything there with the one below.
- Alternatively, replace the entire file with the one from https://paste.bagelnl.my.id/pasta/spider-otter-bird

```json
{
    "OfficialToolsDictionary": {
        "Honkai": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Daily Check-in",
                "URL": "https://act.hoyolab.com/bbs/event/signin-bh3/index.html?act_id=e202110291205111"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "HoYoLab Website",
                "URL": "https://www.hoyolab.com/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Honkai Impact 3rd Wiki",
                "URL": "https://honkaiimpact3.fandom.com/wiki/"
            }
        ],
        "Genshin": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Daily Check-in",
                "URL": "https://act.hoyolab.com/ys/event/signin-sea-v3/index.html?act_id=e202102251931481&hyl_auth_required=true&hyl_presentation_style=fullscreen&utm_source=hoyolab&utm_medium=tools&lang=en-us&bbs_theme=light&bbs_theme_device=1"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "HoYoLab Website",
                "URL": "https://www.hoyolab.com/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Gensin Impact Wiki",
                "URL": "https://wiki.hoyolab.com/pc/genshin/home"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Redeem Codes",
                "URL": "https://genshin.hoyoverse.com/gift"
            }
        ],
        "StarRail": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Daily Check-in",
                "URL": "https://act.hoyolab.com/bbs/event/signin/hkrpg/index.html?act_id=e202303301540311"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "HoYoLab Website",
                "URL": "https://www.hoyolab.com/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Honkai: Star Rail Wiki",
                "URL": "https://wiki.hoyolab.com/pc/hsr/home"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Redeem Codes",
                "URL": "https://hsr.hoyoverse.com/gift"
            }
        ]
    },
    "CommunityToolsDictionary": {
        "Honkai": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Reddit Community",
                "URL": "https://www.reddit.com/r/HonkaiImpact3rd/"
            }
        ],
        "Genshin": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "paimon.moe",
                "URL": "https://paimon.moe/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Enka Network",
                "URL": "https://enka.network/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "seelie.me",
                "URL": "https://seelie.me/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Genshin Optimizer",
                "URL": "https://frzyc.github.io/genshin-optimizer"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Inventory Kamera",
                "URL": "https://github.com/Andrewthe13th/Inventory_Kamera$OpenExternalApp:ApplicationName=Inventory Kamera,ApplicationExecName=InventoryKamera.exe,RunAsAdmin=true$OpenUrlIfCancel"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Kongying Map",
                "URL": "https://yuanshen.site/docs/en/$OpenExternalApp:ApplicationName=Kongying Map,ApplicationExecName=Map.exe,RunAsAdmin=true$OpenUrlIfCancel"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "akasha.cv",
                "URL": "https://akasha.cv"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Genshin Configurator",
                "URL": "https://github.com/Myp3a/GenshinConfigurator/$OpenExternalApp:ApplicationName=Genshin Impact Configurator,RunAsAdmin=true$OpenUrlIfCancel"
            }
        ],
        "StarRail": [
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "pom.moe",
                "URL": "https://pom.moe/"
            },
            {
                "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
                "IconGlyph": "",
                "Text": "Project Yatta (HSR)",
                "URL": "https://hsr.yatta.top/"
            },
            {
              "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
              "IconGlyph": "",
              "Text": "Star Rail Station",
              "URL": "https://starrailstation.com/"
            },
            {
              "IconFontFamily": "ms-appx:///Assets/Fonts/FontAwesomeSolid6.otf#Font Awesome 6 Free Solid",
              "IconGlyph": "",
              "Text": "prydwen.gg",
              "URL": "https://www.prydwen.gg/star-rail/"
            }
        ]
    }
}
```

Thank you for reporting the issue <3

**Comment by Cryotechnic** [comment]
Heyo, for future reference, this is the correct type of issue to make! I'm thinking of maybe eventually making a specific "enhancement" template, which would mean that it would not be considered a bug.

For now, this works!

**Comment by Cryotechnic** [comment]
Hi~

Just letting you know that the order is fixed and the change should be deployed in the next stable version of Collapse, 1.70.12.

This issue will remain open until the build is deployed should you have any additional concerns. Thanks for the report!

**Comment by cupangafk** [comment]
this should be closed because 1.70.12 already released

thank you for your hardwork
